### DISCUSSION TITLE
Welcome to engram-es Discussions!

### DISCUSSION THREAD

**Discussion by binarybottle** [binarybottle/engram-es#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lobo-Feroz** [comment]
Hello there,

I'm Miguel from Spain, I train people in science-based personal development (such as: emotion management, decision making, high performance habits or public speaking). I've got some rusty leftover technical skills from a past life where I did actually code.

I've been always interested in alternatives to qwerty, which always seemed very inefficient to me. Actually I used dvorak-es for a couple of years, though it didn't feel much more efficient to me either. With the arrival of algorithmically designed keyboards, I think we now have a chance to have at least one very efficient spanish keyboard, hopefully the first of many so people can choose. 

I read the preprint paper on the development of engram and I found it very insightful and full of very logical design decisions. Personally, I like Engram the most of all the alternatives I've seen. And considering the proccess is replicable for any language, it seems like an awesome candidate for a spanish kb. And also, right now there is _no_ competition ;)

What can I do? My role is that of a science communicator. Mostly I can research, compile and present information. I can run and maybe hack some python if it's not too complex. Of course testing the layout candidates. I can set up and test the layout in windows and ubuntu. Also I'd like to prepare some training for typing websites, if there are any that allow uploading of custom layout & training.

Finally, I own a medium-sized website (lifehacking.tech) with a little but tight-knit community of "life hackers", so I hope I might be able to encourage people to learn about and use a much more efficient keyboard !

**Reply by binarybottle** [reply]
@Lobo-Feroz -- Thank you very much for your interest in this project, and for your kind offer to help!  Very soon I will need some help evaluating and testing layout candidates. Any help toward implementation in windows and linux, to mirror the implementation in the English engram repo would be most appreciated. Training for typical websites would also be incredible!

**Comment by NickG13** [comment]
Hi guys, I will present myself too! 
I'm Nick Gutiérrez, and I've studied psychology degree and ethology master's in Universitat de Barcelona. I've done a lot of different stuff and projects, but I've always maintained a curious spirit and have always been writing my stuff. Likewise, I started freelance writing in 2018 and never stopped growing. 
In my business, I have to write all day long, and I'm always investigating options for becoming more productive while balancing my personal life. Since discovering the layout world, I've become obsessed with the fact that there wasn't a Spanish layout, so I started researching and after a while I've done a personal layout that I'm training today: [AEIOU](https://github.com/NickG13/AEIOU), which combines the best things I liked from several tests. 
In the process, I've learned a lot. I can give a lot of feedback in the user experience I've got with several tests of layouts. 
We will be waiting for your layout proposals, and I will help whenever possible ;)

**Reply by binarybottle** [reply]
@NickG13 -- Great to have another layout enthusiast on board with eclectic interests!  Looking forward to your feedback when we're further aliong!
